### PR TITLE
modifcations for mobile and resizing

### PR DESCRIPTION
**Pull request by doodlemarks**
Made a whole lot of layout changes to better support mobile. Also, fixed some spacing issues and what not.

### CODE REVIEW

File: src/assets/base.css [hunk 1]
```diff
@@ -0,0 +1,124 @@
+*,::before,::after{
+ margin: 0;
+ box-sizing: border-box;
+}
+
+body {
+ width: 100vw;
+ height: 100vh;
+ overflow: hidden;
```
Comment (natemoo-re) [inline src/assets/base.css]: This should help with scrolling on mobile!

```suggestion
 min-height: 100vh;
 overflow-x: hidden;
```

**Review by natemoo-re — APPROVED** [review]
LGTM! 😍

**Comment by aFuzzyBear** [comment]
Looks amazing to me, 💚🫂👊, I'll merge this once I'm on the computer in the morning dude,  thank you for putting this together @doodlemarks 👊

**Review by aFuzzyBear — APPROVED** [review]
The Style changes works a treat dude! Thank you for this, and updating the artwork aswell, 👊

**Pull request merged**